<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 233, 79);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(0, 0, 0);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-left-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-left-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_11">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_13">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_14">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_16">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_17">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_18">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_19">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_20">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_21">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_22">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_23">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_24">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_25">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_26">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_27">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_28">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_29">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_30">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_31">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_32">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_33">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_34">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_35">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_36">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_37">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_38">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_39">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_40">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_41">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_42">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_43">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_44">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_45">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_46">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_47">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_48">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_49">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_50">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_51">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_52">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_53">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_54">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_55">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_56">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_57">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_58">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_59">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_60">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_61">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_62">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_63">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_64">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_65">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_66">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_67">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_68">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_69">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_70">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_71">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_72">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_73">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_74">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_75">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_76">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_77">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_78">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_79">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_80">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_81">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_82">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_83">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_84">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_85">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_86">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_87">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
